### PR TITLE
DESK-135 connection quality

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

Provide connection quality issue indicator and more detailed connection info.

### Test plan

1. Sign in
2. Go to the device list
3. Click the icon of a device with a red / orange indicator on it
4. You should see the device list with an indication of connection quality

### Ticket

<https://remoteit.atlassian.net/browse/DESK-135>

### CODE REVIEW

**Review by javierdac — APPROVED** [review]
All good

File: frontend/src/components/ConnectionStateIcon/ConnectionStateIcon.tsx [hunk 1]
```diff
@@ -74,17 +84,54 @@ export function ConnectionStateIcon({
...
+        <b className={css.capitalize}>{name}</b>
+        <br />
+        <Icon
+          name="circle"
+          color={device.quality === 'POOR' ? 'danger' : 'warning'}
+          size="bug"
+          type="solid"
+          inlineLeft
+        />
+        Connection quality {device.quality.toLowerCase()}
```
Comment (mauricio0000) [inline frontend/src/components/ConnectionStateIcon/ConnectionStateIcon.tsx:108]: if the `quality` is undefined, it is going to broke the component.

`device.quality.toLowerCase()`

Comment (mauricio0000) [reply frontend/src/components/ConnectionStateIcon/ConnectionStateIcon.tsx:108]: never mind 

you are checking 

`  let showQuality = device?.quality === 'POOR' || device?.quality === 'MODERATE'`

Comment (javierdac) [reply frontend/src/components/ConnectionStateIcon/ConnectionStateIcon.tsx:108]: could it be quality undefined?

Comment (JamieRuderman) [reply frontend/src/components/ConnectionStateIcon/ConnectionStateIcon.tsx:108]: The typescript contract says it should never be undefined... so 🤷‍♂️ hopefully it speaks the truth!  🙂

**Review by mauricio0000 — COMMENTED** [review]
you are right. sorry

**Pull request merged**